ready for review

polinaplohotniuk opened the pull request.

polinaplohotniuk committed:
ready for review

k-asadova commented on tests/test_faq.py:
```diff
@@ -0,0 +1,30 @@
+class TestFaq: # класс тестов для раздела "Вопросы и ответы"
+    # параметризованный тест для проверки соответствия вопросов и ответов
+    @pytest.mark.parametrize("question_locator, answer_locator, expected_answer", [
+        (FaqLocators.QUESTION_0, FaqLocators.ANSWER_0, "Сутки — 400 рублей. Оплата курьеру — наличными или картой."),
```
Можно лучше: тексты ответов хранить в константах, чтобы не перегружать визуально тест

k-asadova commented on tests/test_main_page.py:
```diff
@@ -0,0 +1,46 @@
+    with allure.step("Переключение на новое окно"):
+        main_page.switch_to_new_window()
+    with allure.step("Проверка URL после редиректа"):
+        WebDriverWait(driver, 15).until(
```
Нужно поправить: в подходе РОМ весь код должен быть в степах, "голых" обращений к драйверу быть не должно в тестах

k-asadova commented on tests/test_main_page.py:
```diff
@@ -0,0 +1,46 @@
+    with allure.step("Инициализация главной страницы"):
+        main_page = MainPage(driver)
+    with allure.step("Открытие страницы заказа"):
+        driver.get(SCOOTER_URL + "/order") # открываю страницу заказа
```
Тут тоже степы добавляем

k-asadova commented on tests/test_order.py:
```diff
@@ -0,0 +1,28 @@
+    with allure.step("Инициализация страницы заказа"):
+        page = OrderPage(driver) # создаём экземпляр страницы заказа
+    with allure.step("Открытие страницы Самоката"):
+        page.driver.get(SCOOTER_URL) # открываем страницу Самоката
```
+

polinaplohotniuk committed:
Ready for review again.

Changed the project structure.
Added 'data' folder and moved helpers.py to it.
Added the file 'constants.py'.
Removed from my tests all "naked" interactions with driver.
